check_upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Maybe this script is better in `platform_it/src/test/bin/` similar to `start-splice-its` for instance.

Can you try integrating it into the verification maven goal too? After running the NSDS tests we would shut splice down, download the tarball, unpack it, restart (without building) and run the UpgradeIT test.

Reply:
yes, working on it